include/drivers/dma.h

```diff
@@ -98,25 +99,28 @@ struct dma_block_config {
 /**
  * @brief DMA configuration structure.
  *
- *     dma_slot             [ 0 : 5 ]   - which peripheral and direction
+ *     dma_slot             [ 0 : 6 ]   - which peripheral and direction
```

Comment:
Can you please explain why this grew by one bit?

Reply:
ok, in Imx.rt series the dma slot are 127 so I need  to expand them. I will add it in the commit message